scripts/index.js

```diff
@@ -1,2 +1,13 @@
 // add your solutions here
-console.log("hello")
\ No newline at end of file
+function isPositive(nums){
+    result = []
```

Comment:
Make sure to use `let` or `const` when using a variable for the first time. Otherwise, you might accidentally create a global variable. Check out http://tobyho.com/2011/10/25/js-accidental-global-variables/ for more info.